web-client/src/presenter/actions/TrialSession/validateTrialSessionAction.js

```diff
@@ -16,23 +16,36 @@ export const validateTrialSessionAction = ({
   path,
   props,
 }) => {
-  const startDate = // AAAA-BB-CC
+  const startDate =
```

Comment:
💭 This could potentially also be DRY-d up as part of DOD too

Reply:
Pulled out the shop vac and DRYed up!